event_model/__init__.py

```diff
@@ -1354,3 +1354,103 @@ def default(self, obj):
                 return obj.item()
             return obj.tolist()
         return json.JSONEncoder.default(self, obj)
+
+
+class OrderError(ValueError):
+    ...
+
+
+def validate_order(run_iterable):
+    """
+    Validates the order of a Bluesky Run.
```

Comment:
Can you copy over the content from #98 here to explain what constraints this is enforcing?